deploy-manage/users-roles/custom-roles.md

```diff
@@ -0,0 +1,93 @@
+
+All custom roles grant the same access as the `Viewer` instance access role with regards to {{ecloud}} privileges. To grant more {{ecloud}} privileges, assign more roles. Users receive a union of all their roles' privileges.
+
+Roles are a collection of privileges that enable users to access project features and data. When you create a custom role, you can assign {{es}} [cluster](#custom-roles-es-cluster-privileges) and [index](#custom-roles-es-index-privileges) privileges and [{{kib}}](#custom-roles-kib-privileges) privileges.
+
+On this page, you'll learn about how to [manage custom roles in your project](#manage-custom-roles), the types of privileges you can assign, and how to [assign the roles](#assign-custom-roles) that you create.
+
+::::{note}
+You cannot assign [run as privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/elasticsearch-privileges.md#_run_as_privilege) in {{serverless-full}} custom roles.
+
+::::
+
+## Manage custom roles
+
+You can manage custom roles from within your project, in **{{project-settings}} > {{manage-app}} > {{custom-roles-app}}**. To create a new custom role, click the **Create role** button. To clone, delete, or edit a role, open the actions menu `⋯`.
+
+## {{es}} cluster privileges [custom-roles-es-cluster-privileges]
+
+Cluster privileges grant access to monitoring and management features in {{es}}. They also enable some stack management capabilities in your project.
+
+:::{image} ../../images/serverless-custom-roles-cluster-privileges.png
+:alt: Create a custom role and define {{es}} cluster privileges
+:class: screenshot
+:::
+
+Refer to [cluster privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/elasticsearch-privileges.md#privileges-list-cluster) for a complete description of available options.
+
+
+## {{es}} index privileges [custom-roles-es-index-privileges]
+
+Each role can grant access to multiple data indices, and each index can have a different set of privileges. Typically, you will grant the `read` and `view_index_metadata` privileges to each index that you expect your users to work with. For example, grant access to indices that match an `acme-marketing-*` pattern:
+
+:::{image} ../../images/serverless-custom-roles-index-privileges.png
+:alt: Create a custom role and define {{es}} index privileges
+:class: screenshot
+:::
+
+Refer to [index privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/elasticsearch-privileges.md#privileges-list-indices) for a complete description of available options.
+
+### Document-level and field-level security
+
+Document-level and field-level security affords you even more granularity when it comes to granting access to your data: 
```

Comment:
I'd suggest to include some links to the docs that describe DLS and FLS in detail, for reference purposes, except if we are unsure if the official docs apply to serverless...

Reply:
they currently don't, but should. working on that in the next PR, but can start with a link here.

Reply:
will take care of this in the next PR